raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -102,7 +102,9 @@ contract NettingChannelContract {
     /// @param secret The secret to unlock the locked transfer.
     function unlock(bytes locked_encoded, bytes merkle_proof, bytes32 secret) {
         data.unlock(msg.sender, locked_encoded, merkle_proof, secret);
-        ChannelSecretRevealed(secret);
+
+        // Note: this is only valid as long as msg.sender is one of the participants
```

Comment:
If `msg.sender` is not a participant the line above will throw, so I guess it's fine.